### PR TITLE
Valid golem messages

### PR DESCRIPTION
**Pull request by kbeker**
Branch passed for pull request. After finished golem-messages-in-api-view will need some changes

### CODE REVIEW

File: concent_api/api-integration-test.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python3
+
+import sys
+import datetime
```
Comment (rwrzesien) [inline concent_api/api-integration-test.py]: Please do those alphabetically too.

File: concent_api/api-integration-test.py [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+            else:
+                print('{}{:30} ='.format(' ' * indent, field))
+                print_golem_message(nested_message, indent = indent + 4)
+        else:
+            print('{}{:30} = {}'.format(' ' * indent, field, value))
+
+
+def api_request(host, endpoint, private_key = None, data = None, headers = None, ):
+    assert all(value not in ['', None] for value in [endpoint, host, headers])
+    import ipdb; ipdb.set_trace()
```
Comment (rwrzesien) [inline concent_api/api-integration-test.py]: That is needed here or left after debugging ?

File: concent_api/api-integration-test.py [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+        data = dump(
+            data,
+            private_key,
+            CONCENT_PUBLIC_KEY,
+        )
+
+    if data is None:
+        response = requests.post("{}".format(url), headers = headers)
+    else:
+        response = requests.post("{}".format(url), headers = headers, data = data)
```
Comment (rwrzesien) [inline concent_api/api-integration-test.py]: There is no need to split that to if, you can pass data to requests.post() even if it is None.

File: concent_api/api-integration-test.py [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+        print('MESSAGE:')
+        print_golem_message(decoded_response)
+    else:
+        print('STATUS: {} {}'.format(response.status_code, http.client.responses[response.status_code]))
+        if response.text not in ['', None]:
+            print('RAW RESPONSE: {}'.format(response.text))
+    print()
+
+
+def parse_command_line(command_line):
```
Comment (rwrzesien) [inline concent_api/api-integration-test.py]: @cameel argparse.ArgumentParser() should do the job, but maybe you guys agreed to implement this in more custom way ?

File: concent_api/api-integration-test.py [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+    return cluster_url
+
+
+def force_report_computed_task(task_id, provider_private_key, provider_public_key, requestor_private_key, requestor_public_key, current_time):
+
+    compute_task_def = ComputeTaskDef()
+    compute_task_def['task_id'] = task_id
+    compute_task_def['deadline'] = current_time + 60
+    task_to_compute = TaskToCompute(
+        timestamp = current_time - 3540,
```
Comment (rwrzesien) [inline concent_api/api-integration-test.py]: How about putting this value 3540 in settings ? Or maybe it would never need to be changed, then its ok.

**Comment by kbeker** [comment]
Closing this pull request because of amount changes during last weeks on this branch.

**Pull request closed without merging**